src/pages/DeviceHealthCheck/DeviceHealthCheck.tsx

```diff
@@ -0,0 +1,68 @@
+import { Stack, XStack, YStack } from 'tamagui'
+import LayoutComponent from '../../components/LayoutComponent'
+import NimbusLogo from '../../components/NimbusLogo'
+import Titles from '../../components/Titles'
+import DeviceStorageHealth from '../../components/DeviceStorageHealth'
+import DeviceCPULoad from '../../components/DeviceCPULoad'
+import HealthInfoSection from '../../components/HealthInfoSection'
+import { Button, InformationBox } from '@status-im/components'
+import Icon from '../../components/Icon'
+import DeviceMemory from '../../components/DeviceMemoryHealth'
+import DeviceNetworkHealth from '../../components/DeviceNetworkHealth'
+
+const DeviceHealthCheck = () => {
+  return (
+    <LayoutComponent
+      content={<DeviceHeanlthCheckContent />}
+      rightImageSrc="/background-images/eye-background.png"
+    />
+  )
+}
+
+export default DeviceHealthCheck
+
+const DeviceHeanlthCheckContent = () => {
+  return (
+    <div className="container-inner landing-page">
+      <YStack
+        space={'$4'}
+        style={{
+          justifyContent: 'end',
+          alignItems: 'start',
+          marginBottom: '2rem',
+          maxWidth: '100%',
+        }}
+      >
+        <NimbusLogo />
+        <Titles
+          title="Device Health Check"
+          subtitle="Configure your device to start Staking on Nimbus"
+          isAdvancedSettings={true}
+        />
+        <XStack space={'$4'}>
+          <DeviceStorageHealth storage={44} maxStorage={30} />
+          <DeviceCPULoad load={[12, 123, 4,90]} />
```

Comment:
This seems unformatted? Let's make sure everyone has `prettier` running. There will be a CI build step that enforces this as well.